### PR TITLE
Remove daemon-reload completely

### PR DESCRIPTION
**Pull request by malaupa**
* reason: the script runs not with user env
   and --global seems wrong

### CODE REVIEW

File: build/package/postremove.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ UNIT='fw-id-agent.service'
 case "$1" in
   'remove')
     if [ -d /run/systemd/system ] ; then
-      systemctl --user daemon-reload >/dev/null || true
+      systemctl --global daemon-reload >/dev/null || true
```
Comment (hwipl) [inline build/package/postremove.sh:9]: nit-pick: looking at the docs, it could be sufficient to call `systemctl daemon-reload`.

**Review by hwipl — APPROVED** [review]

File: build/package/postremove.sh (hunk 1 above)
Comment (malaupa) [reply build/package/postremove.sh:9]: This is the question, what is right here... does `systemctl --global daemon-reload` anything other than `systemctl daemon-reload`?

Comment (hwipl) [reply build/package/postremove.sh:9]: Looking at https://github.com/systemd/systemd/blob/main/src/systemctl/systemctl-daemon-reload.c#L16 and https://github.com/systemd/systemd/commit/d77d42ed3ae95ee035dce4707777b077d1a9bf8b it seems like `--global` is wrong and will produce an error in a future systemd version.

Comment (malaupa) [reply build/package/postremove.sh]: I removed every flag. Hopefully usage without any flag is right.

**Review by jandd — APPROVED** [review]

**Pull request merged**